### PR TITLE
Closes #1 Fixes Reading Level

### LINKED ISSUE #1: Create a "Reading Level" module
How can we systematically determine the approximate reading (grade) level of a book given its fulltext?

### PR DESCRIPTION
**Pull request by finnless**
Adds py-readability-metrics and lexile reading levels

### CODE REVIEW

File: bgp/modules/terms.py [hunk 1]
```diff
@@ -61,18 +65,56 @@ def results(self):
...
+        self.readability_fk_score= None
+        self.readability_s_score= None
         self.time = 0
 
     def run(self, doc, **kwargs):
-        import textstat
-        self.flesch_kincaid_grade = textstat.flesch_kincaid_grade(doc)
+        import requests
+        from readability import Readability
+        from readability.scorers.flesch_kincaid import ReadabilityException
```
Comment (mekarpeles) [inline bgp/modules/terms.py:79]: We may want to eventually move these dependencies (especially `requests`) outside of this method

see PEP8:
https://www.python.org/dev/peps/pep-0008/#imports

File: bgp/modules/terms.py [hunk 2]
```diff
@@ -61,18 +65,56 @@ def results(self):
...
+            self.lexile_max_age = str(lexile.json()['data']['work']['max_age'])
+        try:
+            r = Readability(doc)
+            fk = r.flesch_kincaid()
+            s = r.smog()
+            self.readability_fk_score = fk.score
+            self.readability_s_score = s.score
+        # If less than 100 words
+        except ReadabilityException:
+            pass
```
Comment (mekarpeles) [inline bgp/modules/terms.py:99]: This may be something we want to log in our errors (e.g. how many books have < 100 words?)

**Comment by mekarpeles** [comment]
Some comments, but nothing blocking. Will merge and we can treat any of the other comments as new issues (if there's consensus any of them are significant enough to merit such.

Thanks for your hard work incorporating reading-levels!

File: bgp/modules/terms.py (hunk 1 above)
Comment (finnless) [reply bgp/modules/terms.py:79]: This is addressed in Move dependencies to top of file #48

**Pull request merged**